### PR TITLE
Feature/get parent affiliates

### PR DESCRIPTION
**Pull request by Jrobles70**
### Fixed
 - Patch tests that were failing
 - Core URL

 ### Added
 - Auth class
 - Scope property for Client and Affiliate
 - Parent Scope property for Affiliate

### CODE REVIEW

File: ticketguardian/affiliate/affiliate.py [hunk 1]
```diff
@@ -18,3 +18,27 @@ def parent(self):
...
+    @property
+    def scope(self):
+        """
+        Used to gets all children within the scope of this Affiliate.
+        Returns:
+            list: a list of all children of the Affiliate
+        """
+        return self.retrieve(self.id, 'scope', raw_data=True)
+
+    def get_parent_scope(self):
```
Comment (austinphilp) [inline ticketguardian/affiliate/affiliate.py]: Why is this one formatted as a `get_...` method while the other is a property? we should keep it consistent, I'm fine doing it either way

Comment (Jrobles70) [reply ticketguardian/affiliate/affiliate.py]: Ok good point I just updated it right now.

**Comment by austinphilp** [comment]
Looks good! Lets make sure that the NOTA release does not contain any references to the old scoping methods though, I think it's still using `get_parent_scope` for instance

**Comment by Jrobles70** [comment]
@austinphilp I just noticed and fix that a few minutes ago actually 👍

**Pull request merged**